_data/projects.yml

```diff
@@ -110,6 +110,95 @@
     - js
     - bulma
 
+- name: Playnet
+  wip: "true"
```

Comment:
Quick comment: YAML supports booleans, which are slightly different than the `"true"` string. I would change this (and the other instances) to:

```yaml
wip: true
```

This is a simple ctrl/cmd-f, so hopefully not too bad!